### ISSUE TITLE
Return code is incorrect on violation

### ISSUE THREAD

**Issue by pocc** [oclint/oclint#538]
### Issue Summary
When there are errors with the code, oclint's code should be non-zero. This is important for meta-linters that depend on the return code.

### Environment
Macos 10.14
```
bash-5.0$ oclint --version
LLVM (http://llvm.org/):
  LLVM version 5.0.0svn-r313528
  Optimized build.
  Default target: x86_64-apple-darwin18.5.0
  Host CPU: haswell

OCLint (http://oclint.org/):
  OCLint version 0.13.
  Built Sep 18 2017 (08:58:40).
```

Installed with `brew install oclint`

### Reproduction Steps / Sample code

```
int main() { int i; return 10; }
```

Output for this line:

```
bash-5.0$ oclint bad.c


OCLint Report

Summary: TotalFiles=1 FilesWithViolations=1 P1=0 P2=0 P3=2

.../bad.c:1:14: short variable name [naming|P3] Length of variable name `i` is 1, which is shorter than the threshold of 3
.../bad.c:1:14: unused local variable [unused|P3] The local variable 'i' is unused.

[OCLint (http://oclint.org) v0.13]
bash-5.0$ echo $?
0
```

### Expected Behavior
Expected behavior is to return 1 or greater error code.

### Actual Behavior
Actual behavior is to return error code of 0 on errors.

**Comment by ryuichi-assistant** [comment]
There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues, as many of them have already been resolved with the latest updates.

Please make sure to update to the latest version and check if that solves the issue. Let us know if that works for you by adding a comment.

**Comment by pocc** [comment]
I have verified that this issue persists on v20.11 on Ubuntu 20.20.

**Comment by ryuichis** [comment]
@pocc If you specify the `-max-priority-3` to `0`, will the tool behaves as you expect? Additional info can be found at https://docs.oclint.org/en/stable/manual/oclint.html#exit-status-options

**Comment by pocc** [comment]
@ryuichis Thank you so much for your reply. Using this option, we get a return code of 5, which solves this issue.
